foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model/OpenAiChatCompletionOutput.java

```diff
@@ -25,4 +26,16 @@ public class OpenAiChatCompletionOutput extends OpenAiCompletionOutput {
   @JsonProperty("system_fingerprint")
   @Getter(onMethod_ = @Nonnull)
   private String systemFingerprint;
+
+  /**
+   * Get the message content from the output.
+   *
+   * <p>Note: If there are multiple choices only the first one is returned
+   *
+   * @return the message content or empty string.
+   */
+  @Nonnull
+  public String getContent() {
+    return Objects.requireNonNullElse(getChoices().get(0).getMessage().getContent(), "");
```

Comment:
Please also check for `getChoises() != null` and `get(0) != null` and `getMessage() != null`

Reply:
They're all non null by the spec

Reply:
I'm not so sure we should offer this without throwing on content filter hits. Users will likely not check for `getChoices().get(0).getFinishReason()` when using `getContent()`